### PR TITLE
Update the status text when the session is paused

### PR DESCRIPTION
**Pull request by leafty**
Screenshot 2023-08-29 at 11 22 15
Screenshot 2023-08-29 at 11 23 43

### CODE REVIEW

File: client/src/features/session/components/status/SessionStatusText.tsx [hunk 1]
```diff
@@ -40,13 +42,38 @@ export default function SessionStatusText({
...
 
   return status === "running" ? (
     <>Running, created {startTimeText}</>
   ) : status === "starting" ? (
     <>Starting, created {startTimeText}</>
   ) : status === "stopping" ? (
     <>Deleting...</>
+  ) : status === "hibernated" && hibernationCullTimestamp ? (
+    <>
+      Paused, will be deleted{" "}
```
Comment (lorenzo-cavazzi) [inline client/src/features/session/components/status/SessionStatusText.tsx:70]: We don't need to repeat the status here. From you screenshot, the "Paused" word is already written one line above this sentence. I would go for something shorter like
```suggestion
      Deleting in{" "}
```

File: client/src/features/session/components/status/SessionHibernationStatusDetails.tsx [hunk 2]
```diff
@@ -43,6 +62,16 @@ export default function SessionHibernationStatusDetails({
               noCaption
             />
           </p>
+          {hibernationCullTimestamp && (
+            <p className="mb-0">
+              <span className="fw-bold">Will be deleted:</span>{" "}
```
Comment (lorenzo-cavazzi) [inline client/src/features/session/components/status/SessionHibernationStatusDetails.tsx:67]: To keep it consistent with the other suggestion
```suggestion
              <span className="fw-bold">Deleting in </span>{" "}
```

Comment (lorenzo-cavazzi) [reply client/src/features/session/components/status/SessionHibernationStatusDetails.tsx:67]: Oh wait, is the time absolute here?

**Review by lorenzo-cavazzi — CHANGES_REQUESTED** [review]

File: client/src/features/session/components/status/SessionStatusText.tsx (hunk 1 above)
Comment (leafty) [reply client/src/features/session/components/status/SessionStatusText.tsx:70]: "Paused" is not written in the first screenshot.

Comment (lorenzo-cavazzi) [reply client/src/features/session/components/status/SessionStatusText.tsx:70]: image

Comment (leafty) [reply client/src/features/session/components/status/SessionStatusText.tsx:70]: This one: Screenshot 2023-08-29 at 11 22 15

Comment (lorenzo-cavazzi) [reply client/src/features/session/components/status/SessionStatusText.tsx:70]: Ok, on the sessions tab it makes sense to keep `Paused` at the beginning.
I still believe the message on the dashboard is unnecessarily long and can be abbreviated as suggested, but I'm ok with keeping it as is

**Review by lorenzo-cavazzi — APPROVED** [review]

**Pull request merged**